### DISCUSSION TITLE
saving settings with 2 accounts

### DISCUSSION THREAD

**Discussion by hjmille** [runelite/runelite#17047] [Q&A] [answered]
good afternoon all, is there a way to save settings for 2 seperate accounts when launching with jagex launcher? If so, is there a tutorial on how to do it that can be linked? tyvm!!

**Comment by geheur** [comment]
you have to manually change the profile after starting the client. There is no `--profile` option that works with jagex launcher.

**Reply by hjmille** [reply]
thank you! i see where to do it now

**Comment by Nightfirecat** [comment, marked as answer]
There is not. If merged, #16749 would make it possible to have automatic profile switching for multiple Jagex-launcher-launched clients.

**Reply by hjmille** [reply]
thank you so much for the quick reply. so this would save settings to each account and when selected, would just load the profile? sorry, i am new to most of this.

**Reply by Nightfirecat** [reply]
It's not added in the client yet; there will be instructions available for it on the wiki (and blog post) if it is. Until then, it's not available as an option yet.

**Reply by hjmille** [reply]
thank you, i see on RL where i can swap over manually. youre the best
